bugnote_view_inc.php

```diff
@@ -155,7 +155,7 @@
 ?>
 <tr class="bugnote" id="c<?php echo $t_activity['id'] ?>">
 		<td class="category">
-		<div class="pull-left padding-2"><?php print_avatar( $t_activity['user_id'] ); ?>
+		<div class="pull-left padding-2"><?php print_avatar( $t_activity['user_id'], 'bugnote_view_inc', 80 ); ?>
```

Comment:
@syncguru vboctor told me to do it this way:
>     Update print_avatar to accept a CSS class prefix. Then we can add to it _(size), then _container and _avatar. This way we have consistent naming pattern and we can provide default for the CSS prefix.
Can you state on this before I push a new commit? Doing it this way allows to use the function in different places (it's not the case now but it could be if you plan to add a little avatar in the bug detail page for the reporter and assignee).